skins/oasis/css/core/breakpoints-layout.scss

```diff
@@ -318,3 +314,59 @@
 		}
 	}
 }
+
+// Visual Editor changes for breakpoints
+$ve-overlay-color: rgba(0, 0, 0, .5);
+$vw-overlay-shadow-size: 100vw; // IE does not support vmax
+body.ve {
+	pointer-events: none;
+
+	.ve-ui-overlay {
+		pointer-events: all;
+	}
+
+	// do not show VE overlay
+	.ve-ui-wikiaFocusWidget {
```

Comment:
if we are not showing it please create a ticket for clean up to remove this thing